src/doc/rustdoc/src/the-doc-attribute.md

```diff
@@ -214,3 +214,31 @@ the `strip-hidden` pass is removed.
 Since primitive types are defined in the compiler, there's no place to attach documentation
 attributes. This attribute is used by the standard library to provide a way to generate
 documentation for primitive types.
+
+## `#[cfg(rustdoc)]`: Documenting platform-/feature-specific information
+
+Because of the way Rustdoc documents a crate, the documentation it creates is specific to the target
+rustc compiles for. Anything that's specific to any other target is dropped via `#[cfg]` attribute
+processing early in the compilation process. However, Rustdoc has a trick up its sleeve to handle
+platform-specific code if it *does* receive it.
+
+For Rustdoc to document an item, it needs to see it, regardless of what platform it's currently
+running on. To aid this, Rustdoc sets the flag `#[cfg(rustdoc)]` when running on your crate.
+Combining this with the target platform of a given item allows it to appear when building your crate
+normally on that platform, as well as when building documentation anywhere.
+
+For example, `#[cfg(any(windows, rustdoc))]` will preserve the item either on Windows or during the
+documentation process. For example:
+
+```rust
+/// Token struct that can only be used on Windows.
+#[cfg(any(windows, all(windows, rustdoc)))]
```

Comment:
These samples should just use `#[cfg(any(platform, rustdoc))]`, to match the text above it. As written, they will still only appear when generating the docs using a `--target` for their selected platform. (The current cfg attributes are like, "this should only appear *either* on Windows/Unix, *or* when *both* Windows/Unix and `rustdoc` are set".)

Reply:
Agreed, that even simplifies the example. Updating it.